modules/lib.lua

```diff
@@ -356,19 +366,38 @@ else
     ---@class LibDrawText3DParams : LibDrawTextParams
     ---@field coords vector3
     ---@field disableDrawRect? boolean
+    ---@field scale? vector2 default: `vec2(0.35,0.35)`
+    ---@field enableDropShadow? boolean 
+    ---@field enableOutline? boolean 
 
     ---Draws text onto the screen in 3D space for a single frame.
     ---@param params LibDrawText3DParams
     function qbx.drawText3d(params) -- luacheck: ignore
+        local isScaleparamANumber = type(params.scale) == "number"
         local text = params.text
         local coords = params.coords
-        local scale = params.scale or 0.35
+        local scale = (isScaleparamANumber and vec2(params.scale, params.scale))
+                  or params.scale
+                  or vec2(0.35, 0.35)
         local font = params.font or 4
         local color = params.color or vec4(255, 255, 255, 255)
+        local enableDropShadow = params.enableDropShadow or false
+        local enableOutline = params.enableOutline or false
+        if isScaleparamANumber and not hasText3dScaleAsNumberDepricatedMessageShown then
+            lib.print.warn('This resource is still using qbx.drawText3d with a number as the scale parameter!')
+            lib.print.warn('If you are the author, please update to use a vec2 for the scale parameter. If you are not, please tell them to update!')
```

Comment:
Let's remove this. I don't think we need to force people to update their scripts over this.

Reply:
Done